### PR TITLE
Website and climtemp bug fix

### PR DESCRIPTION
**Pull request by maxwellfundi**

### CODE REVIEW

File: instat/dlgClimaticCheckDataTemperature.vb [hunk 1]
```diff
@@ -598,14 +598,14 @@ Public Class dlgClimaticCheckDataTemperature
...
 
     Private Sub CoreControls_ControlContentsChanged(ucrChangedControl As ucrCore) Handles ucrReceiverElement1.ControlContentsChanged, ucrReceiverElement2.ControlContentsChanged, ucrNudSame.ControlContentsChanged, ucrNudRangeElement1Min.ControlContentsChanged, ucrNudRangeElement1Max.ControlContentsChanged, ucrNudRangeElement2Min.ControlContentsChanged, ucrNudRangeElement2Max.ControlContentsChanged, ucrNudJump.ControlContentsChanged, ucrNudRangeElement2Min.ControlContentsChanged, ucrNudRangeElement2Max.ControlContentsChanged, ucrNudDifference.ControlContentsChanged, ucrChkRangeElement1.ControlContentsChanged, ucrChkRangeElement2.ControlContentsChanged, ucrChkJump.ControlContentsChanged, ucrChkDifference.ControlContentsChanged, ucrChkSame.ControlContentsChanged, ucrChkOutlier.ControlContentsChanged
         TestOkEnabled()
     End Sub
 
-    Private Sub ucrReceiverMonth_ControlValueChanged(ucrChangedControl As ucrCore) Handles ucrReceiverMonth.ControlValueChanged
+    Private Sub ucrReceiverMonth_ControlValueChanged(ucrChangedControl As ucrCore) Handles ucrReceiverMonth.ControlValueChanged, ucrReceiverElement1.ControlValueChanged, ucrReceiverElement2.ControlValueChanged
```
Comment (dannyparsons) [inline instat/dlgClimaticCheckDataTemperature.vb]: why this change?

Comment (maxwellfundi) [reply instat/dlgClimaticCheckDataTemperature.vb]: The groupbymonth function parameters were not being updated properly when the two receivers we added changed. @volloholic  Is aware of this issue and he was happy with this fix.

Comment (dannyparsons) [reply instat/dlgClimaticCheckDataTemperature.vb]: Ok sounds good. Better to do it on a separate branch though.

Comment (maxwellfundi) [reply instat/dlgClimaticCheckDataTemperature.vb]: yeah I agree!
